src/main/flight/pid.c

```diff
@@ -136,10 +136,21 @@ typedef float (*pidControllerFn)(const pidProfile_t *pidProfile, int axis, float
 float featheredPids(const pidProfile_t *pidProfile, int axis, float errorRate, float dynCi, float currentPidSetpoint);
 float classicPids(const pidProfile_t *pidProfile, int axis, float errorRate, float dynCi, float currentPidSetpoint);
 
+float errorBoostLimit = 1.0f;
+float errorMultiplier = 1.0f;
+
+float featheredPIDboost( float errorRate)
+{
+	float boostedErrorFactor = (errorRate * errorRate) * errorMultiplier;
+	boostedErrorFactor += 1.0f;
+	boostedErrorFactor = MIN(boostedErrorFactor, errorBoostLimit);
+
+	return boostedErrorFactor;
+}
 
 void resetPidProfile(pidProfile_t *pidProfile)
 {
-	r_weight = 0.67;
+	r_weight = 0.33;
```

Comment:
Why change the r_weight?

Reply:
that line can be removed.
33 is the default imu-f value.

Reply:
ok I didn't realize that.
I'll get some people testing their rweight at .33 and see what they think of it.
I think we should wait for now before doing that change though, but that's good info to have.